### PR TITLE
Story3 single random number

### PR DESCRIPTION
**Pull request by Joshsilas**
This is Task 1 and Task 2 as both needed to be completed for working Purposes

### CODE REVIEW

File: src/generateNumberCards.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+function randomInt(max, min) {
+    return Math.floor(Math.random() * (max - min + 1) + min)
+}
+
+function createMultipleCardsBetween3And5(){
```
Comment (CMeier556) [inline src/generateNumberCards.js]: Now that you have the functionality working, maybe change this function to createCards or something simple?

Comment (CMeier556) [reply src/generateNumberCards.js]: Maybe not the best idea but a thought

File: src/generateNumberCards.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            randomNumber = randomInt(100, 0)
+        } 
+        randomNumbersHolder[i] = randomNumber
+    }
+
+    Object.values(randomNumbersHolder).forEach(number => {
+        let content = document.querySelector('.number-cards-container').innerHTML
+        content +=  
+            `<div class="number-card">
+                <p>${number}</p>
```
Comment (CMeier556) [inline src/generateNumberCards.js]: maybe there is a way to use textContent and put the HTML in the HTML document

File: src/generateNumberCards.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+function randomInt(max, min) {
+    return Math.floor(Math.random() * (max - min + 1) + min)
+}
+
+function createMultipleCardsBetween3And5(){
+let randomNumbersHolder = {}
+    for (let i = 0; i < randomInt(5,3); i++) {
+        let randomNumber = randomInt(100, 0)
+        while (Object.values(randomNumbersHolder).includes(randomNumber)) {
+            randomNumber = randomInt(100, 0)
```
Comment (CMeier556) [inline src/generateNumberCards.js]: Since you already set the variable, you may not need the '=randomInt(100,0)' when executing the loop. Not sure though

**Review by CMeier556 — CHANGES_REQUESTED** [review]

File: src/generateNumberCards.js (hunk 2 above)
Comment (alexjames4) [reply src/generateNumberCards.js]: I think it is needed

File: src/generateNumberCards.js (hunk 3 above)
Comment (alexjames4) [reply src/generateNumberCards.js]: I think we need it for the while loop to work

**Review by CMeier556 — APPROVED** [review]

**Pull request merged**